docs/getting-started/mainnet.md

```diff
@@ -0,0 +1,469 @@
+
+Once the node is synced, we are now ready to send a `create-validator` transaction and join the network, for example:
+
+```
+$ ./chain-maind tx staking create-validator \
+--from=[name_of_your_key] \
+--amount=[amount of cro, i.e. 1000cro] \
+--pubkey=[crocnclconspub...]  \
+--moniker="[The_id_of_your_node]" \
+--security-contact="[security contact email/contact method]" \
+--chain-id="crypto-org-chain-mainnet-1" \
+--commission-rate="0.10" \
+--commission-max-rate="0.20" \
+--commission-max-change-rate="0.01" \
+--min-self-delegation="1" \
+--gas 80000000 \
+--gas-prices 0.1basecro
+
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator"...}
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+
+
+  ::: tip You will be required to insert the following:
+  - `--from`: The `cro...` address or the key name that holds your funds for initial delegation;
+  - `--pubkey`: The validator public key ( See Step [3-2](#step-3-2-obtain-the-validator-public-key) above ) with **crocnclconspub** as the prefix;
+  - `--moniker`: A moniker (name) for your validator node;
+  - `--security-contact`: Security contact email/contact method, it is **strongly recommended** to provide an email address for receiving important messages related to validator operation in the future;
+  - `--commission-rate`: The commission rate charge on the delegator;
+  - `--commission-max-rate`: The upper bound of the commission rate;
+  - `--commission-max-change-rate`: The maximum daily increase of the validator commission. Please note this parameter cannot be changed after create-validator is processed.
+  :::
+
+
+### Step 3-6. Check your validator status
+Once the `create-validator` transaction completes, you can check if your validator has been added to the validator set:
+
+```bash
+$ ./chain-maind tendermint show-address
+## [crocnclcons... address] ##
+$ ./chain-maind query tendermint-validator-set | grep -c [crocnclcons...]
+## 1 = Yes; 0 = Not yet added ##
+```
+
+To further check if the validator is signing blocks, kindly run this [script](https://github.com/crypto-com/chain-docs/blob/master/docs/getting-started/assets/signature_checking/check-validator-up.sh), for example:
+
+```bash
+$ curl -sSL https://raw.githubusercontent.com/crypto-org-chain/chain-docs/master/docs/getting-started/assets/signature_checking/check-validator-up.sh | bash -s -- \
+--tendermint-url https://mainnet.crypto.org:26657 \
+--pubkey $(cat ~/.chain-maind/config/priv_validator_key.json | jq -r '.pub_key.value')
+
+The validator is in the active validator set under the address  <YOUR_VALIDATOR_ADDRESS>
+The validator is signing @ Block#<BLOCK_HEIGHT> 👍
+```
+
+
+
+Alternatively, you can run it on this [browser based IDE](https://repl.it/@allthatjazzleo/cryptocomcheckNodeJoinStatus#main.go), by specifying your validator public key in the `"YOUR_PUBKEY"` field, where this key can be obtained by running
```

Comment:
Is the script pointing to mainnet or testnet now?

Reply:
I planed to remove this browser based IDE version of the signature checking script.